scripts/prepare-release

```diff
@@ -47,10 +45,9 @@ fi
 git fetch origin master
 git switch -C "topic/release-${new_version}" origin/master
 
-# Change the fallback version.
-perl -i -pe "s/${last_rc_version}/${new_version}/g" \
-  "${source_dir}/cmake/VASTVersionFallback.cmake" \
-  "${source_dir}/nix/vast/default.nix"
+# Update the fallback version.
+version_json="$(jq --arg v "${new_version#v}" '."vast-version-fallback" = $v' version.json)"
+echo -E "${version_json}" > version.json)
```

Comment:
Is there a reason for not just piping the output of jq to version.json here?

Reply:
I assume you mean redirecting the output with `>`. The problem is that this operator clears the contents of the file it writes to, and that can happen before `jq` gets around to read the input. That is why we write it to an intermediate variable first.

Reply:
Instead of passing version.json as a positional argument to jq you can also pass it via stdin, i.e., `jq ... < version.json > version.json`, which should guarantee the ordering we want to have.

Reply:
Can you provide a source for that?

Reply:
Apparently that's a Zsh extension I was using for so long, I thought of it being standard.

You could still use `sponge` from moreutils, but I don't know how widely that is available.

Feel free to resolve.

Reply:
I'd rather avoid depending on moreutils, it is usually not part of a base install.